src/main/java/io/confluent/connect/elasticsearch/DataConverter.java

```diff
@@ -115,6 +118,14 @@ public IndexableRecord convertRecord(
     if (record.value() == null) {
       switch (behaviorOnNullValues) {
         case IGNORE:
+          if (log.isTraceEnabled()) {
+            log.trace(
+                "Ignoring record with null value at topic '{}', partition {}, offset {}",
+                record.topic(),
```

Comment:
I think these getters are relatively cheap, and the real cost for this log message (toString and writing to the logger) is already guarded internally.
I added this guard to all the trace calls in my testing, and didn't notice a significant change in throughput, so I think this may be some premature optimization.

Disabling the cyclomatic complexity check and decreasing the overall code readability is probably not worth the microsecond saved here.

Reply:
Fixed.